h1st/model/fuzzy_logic/fuzzy_logic_rules.py

```diff
@@ -0,0 +1,100 @@
+import logging
+from typing import NoReturn
+
+import skfuzzy
+from skfuzzy import control as ctrl
```

Comment:
Use `skctrl` instead of `ctrl`, so that the code reads clearly as coming from SK.